src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -439,7 +450,17 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
           }
         }
       }
+
+    // how to add queries?
```

Comment:
See https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala#L72 for how I dealt with stubbing out different query responses. With those stubbed out, you can make correlated GETs and ensure that you're getting back what you want.